assets/js/updaters.js

```diff
@@ -269,11 +269,11 @@ function updateVoidTraderInventory() {
       /* eslint-disable prefer-template */
       const inventoryString = `${'<div class="panel panel-primary voidTraderInventory" ' +
                 'style="margin-left:5%; margin-right:5%" ' +
-                'id="'}${worldState.voidTrader.id}">\n<div class="panel-heading">\n` +
+                'id="'}${worldState.voidTrader.id}">\n` +
+                '<a href="#voidTraderInventoryPanel" data-toggle="collapse" class="panel-heading show">\n' +
                 `<h3 class="panel-title">${worldState.voidTrader.character} Inventory` +
-                '<a href="#voidTraderInventoryPanel" data-toggle="collapse">' +
-                '<span class="glyphicon glyphicon-triangle-bottom pull-right"></span></a></h3>\n' +
-                '</div>\n' +
+                '<span class="glyphicon glyphicon-triangle-bottom pull-right"></span></h3>\n' +
+                '</a>\n' +
                 '<div class="panel-body collapse" id="voidTraderInventoryPanel">\n' +
                 '<table class="table table-striped table-hover ">\n' +
```

Comment:
looks like the headings on the table are a tad misalinged on mobile resolutions... If you see a fix for it, could you add that?